src/components/StatusBadge/index.tsx

```diff
@@ -14,94 +15,18 @@ interface StatusBadgeProps {
   is4k?: boolean;
   inProgress?: boolean;
   plexUrl?: string;
-  plexUrl4k?: string;
+  serviceUrl?: string;
```

Comment:
What about adding a `url` prop instead that is just used whenever it's present? That makes the API usage clearer I feel.

Reply:
But we display the Plex vs. service URL conditionally based on the status. If we only have a single URL prop, we'd end up repeating that conditional logic everywhere we show a status badge?

Reply:
It would probably make sense to add a `url` prop to the `Badge` component though... 🤔

Reply:
Ah, yeah, maybe on the Badge component then.

Reply:
Already done in aa6029ac2a887e43663391fdf4c4fab2ad87feef 😉